webtau-feature-testing/examples/scenarios/rest/springboot/customerCrud.groovy

```diff
@@ -18,7 +18,11 @@ scenario("CRUD operations for customer") {
         lastName.should == changedLastName // specifying body is optional
     }
 
+    def changedFirstName = "NFN"
```

Comment:
We used to call `get` right after `put` to make sure it took effect. Now we may hide the error in  `put`

Reply:
good point. Let me patch that test up ;-)